src/components/CategoryFilter.js

```diff
@@ -0,0 +1,21 @@
+export const catFilt = [
+  { id: 0, all: 'All' },
+  { id: 1, action: 'Action' },
+  { id: 2, biography: 'Biography' },
+  { id: 3, history: 'History' },
+  { id: 4, horror: 'Horror' },
+  { id: 5, kids: 'Kids' },
+  { id: 6, learning: 'Learning' },
+  { id: 7, sciFi: 'Sci-fi' },
+];
+
+export const filterOpt = [
+  { id: 0, name: 'All' },
+  { id: 1, name: 'Action' },
+  { id: 2, name: 'Biography' },
+  { id: 3, name: 'History' },
+  { id: 4, name: 'Horror' },
+  { id: 5, name: 'Kids' },
+  { id: 6, name: 'Learning' },
+  { id: 7, name: 'Sci-fi' },
+];
```

Comment:
- [ ]  This should be a presentational **component** see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/presentationalCategoryFilter.png) from Pathwright
- [ ] It should **render a select** with categories see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/RenderSelectCaegoryBox.png) from Pathwright
- [ ] It should merge All select option with available categories see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/AllReturnsAllCategories.png) from Pathwright
- [ ] It should be rendered in BookList container